config/ember-try.js

```diff
@@ -7,23 +7,16 @@ module.exports = async function () {
     usePnpm: true,
     scenarios: [
       {
-        name: 'ember-lts-3.20',
+        name: 'ember-lts-4.12',
         npm: {
           devDependencies: {
-            'ember-source': '~3.20.5',
-          },
-        },
-      },
-      {
-        name: 'ember-lts-3.24',
-        npm: {
-          devDependencies: {
-            'ember-source': '~3.24.3',
+            'ember-source': '~4.12',
           },
         },
       },
       {
         name: 'ember-release',
+        allowedToFail: true,
```

Comment:
Will relax this later, after adding ember 5.12 try scenario. Need to update a lot of deps before ember 6 try scenario will work